docs/source/tutorial/trainer.rst

```diff
@@ -72,28 +72,28 @@ components needed for training. The main components are shown below.
 .. image:: ../../image/trainer/trainer.png
 
 Basically, all you need to pass to :class:`~chainer.training.Trainer` is an
-:class:`~chainer.training.Updator`. However, :class:`~chainer.training.Updator`
+:class:`~chainer.training.updater`. However, :class:`~chainer.training.updater`
```

Comment:
It looks better to capitalize all updater in chainer.training.Updater, chainer.training.StandardUpdater , chainer.training.ParallelUpdater